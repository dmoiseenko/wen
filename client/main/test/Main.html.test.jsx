import React from 'react';
import { shallow } from 'enzyme';

import MainHtml from '../Main.html';


it('should render', () => {
  const env = process.env.NODE_ENV;

  process.env.NODE_ENV = 'production';

  const props = {
    root: 'root',
    assets: {
      app: {
        css: 'css'
      },
      manifest: {
        js: 'manifest'
      },
      vendor: {
        js: 'vendor'
      }
    },
    state: 'state'
  };

  const wrapper = shallow(
    <MainHtml {...props} />
  );

  expect(wrapper.node).toMatchSnapshot();

  process.env.NODE_ENV = env;
});
