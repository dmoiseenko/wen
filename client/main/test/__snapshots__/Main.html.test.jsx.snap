// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render 1`] = `
<html
  lang="en"
>
  <head>
    <meta
      charSet="utf-8"
    />
    <meta
      content="IE=edge"
      httpEquiv="X-UA-Compatible"
    />
    <meta
      content="width=device-width, initial-scale=1"
      name="viewport"
    />
    <title>
      Wen
    </title>
    <link
      href="/favicon.ico"
      rel="shortcut icon"
      type="image/x-icon"
    />
    <link
      href="/favicon.ico"
      rel="icon"
      type="image/x-icon"
    />
    <link
      href="css"
      rel="stylesheet"
      type="text/css"
    />
  </head>
  <body>
    <div
      dangerouslySetInnerHTML={
        Object {
          "__html": "root",
        }
      }
      id="react-app"
    />
    <script
      dangerouslySetInnerHTML={
        Object {
          "__html": "window.__APOLLO_STATE__=\\"state\\";",
        }
      }
    />
    <script
      src="manifest"
      type="text/javascript"
    />
    <script
      src="vendor"
      type="text/javascript"
    />
    <script
      src={undefined}
      type="text/javascript"
    />
  </body>
</html>
`;
